src/Compilers/Test/Core/Platform/CoreClr/CoreCLRRuntimeEnvironment.cs

```diff
@@ -84,7 +85,7 @@ public int Execute(string moduleName, string[] args, string expectedOutput, bool
         {
             var emitData = GetEmitData();
             emitData.RuntimeData.ExecuteRequested = true;
-            var (exitCode, output) = emitData.LoadContext.Execute(GetMainImage(), args, expectedOutput?.Length);
+            var (exitCode, output) = TestHelpers.WithCulture(() => emitData.LoadContext.Execute(GetMainImage(), args, expectedOutput?.Length));
```

Comment:
>TestHelpers.WithCulture [](http://example.com/codeflow?start=37&length=23)

If I understand correctly, here we are changing UI culture to "local" culture while a test code, which is compiled by a unit-test, is executed. I guess, I am not sure what aspect we are trying to test by making this culture change. Do we really care if the test scenario is not "global ready"? It feels like we could safely use Invariant culture instead and make it easier to author unit-tests by taking away the worry about ```Console.Write*``` formatting. Then majority of the fixups done in unit-tests could be reverted, I think.

Reply:
I know, I questioned this place, when ```InvariantCulture``` was used here, before. However, that was mostly driven by the lack of understanding what effect that was going to have, and why there is a difference between Desktop vs. CoreClr.